unison-src/transcripts/names.md

```diff
@@ -0,0 +1,20 @@
+ Example uses of the `names` command and output
+```ucm:hide
+.> builtins.mergeio
```

Comment:
Avoid using `builtins.mergeio` where possible, as it just adds unnecessary lag at startup.

In this case, you can just use `builtins.merge`, or even better: just `alias.type ##Int .builtins.Int`